### PR TITLE
Add download link when an order have 2 product type virtual and normal

### LINKED ISSUE #25206: Customer can't download virtual product if order contain a normal product
#### Describe the bug

Customer can't download virtual product if order contain also a normal product

#### Steps to Reproduce

1. Enable SAV 14 days 
2. Make an order with a virtual product + dowload file and with a normal product
3. Set order delivered
4. Go to order history an check if the download link is present 

**Screenshots**

**Possible regession ?** 

Acutaly 

Capture d’écran de 2021-07-01 16-54-43

Needed

Capture d’écran de 2021-07-01 16-55-46

#### Additional information

* PrestaShop version: 178
* PHP version: 7..3

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Customer can't download virtual product if order also contain a normal product.
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/25206
| How to test?      | See issue and follow step to reproduce
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25210)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]
LGTM 👍

**Review by Progi1984 — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

File: themes/classic/templates/customer/_partials/order-detail-return.tpl [hunk 1]
```diff
@@ -57,6 +57,9 @@
               {if $product.product_reference}
                 {l s='Reference' d='Shop.Theme.Catalog'}: {$product.product_reference}<br/>
               {/if}
+              {if isset($product.download_link)}
+              <a href="{$product.download_link}">{l s='Download' d='Shop.Theme.Actions'}</a><br/>
```
Comment (PierreRambaud) [inline themes/classic/templates/customer/_partials/order-detail-return.tpl]:
```suggestion
                <a href="{$product.download_link}">{l s='Download' d='Shop.Theme.Actions'}</a><br/>
```

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @okom3pom 

Thanks for your PR!
Merchandise return **disabled** => OK, I see the link

image

Merchandise return **enabled** => OK, I see the link

image

But I can see in your screenshot, there's no checkbox for virtual product to return, mean you fix this issue #16275

image

But why you didn't share your fix in your PR?
On mobile, there's no link:

image

Please check and feedback.

Thanks!

**Comment by ghost** [comment]
Hello @hibatallahAouadni 

There are two issues and two PR

One for dowload link one for return : https://github.com/PrestaShop/PrestaShop/pull/25204

Sorry i used the branch with the `return product fix` for my screenshots :( 

I added the download link also for mobile.

Thomas

**Comment by hibatallahAouadni** [comment]
Hello @okom3pom 

Thanks again for your PR!
LGTM, QA :heavy_check_mark: 

https://drive.google.com/file/d/1Xfj_9LCwayxdxAtCWtO_vmQf-HlsZtWF/view

Thanks!

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @okom3pom & @hibatallahAouadni

**Pull request merged**